internal/config/config.go

```diff
@@ -215,9 +215,19 @@ type Instrumentation struct {
 
 // SmartRetry represents the settings for retry strategy.
 type SmartRetry struct {
+	FailedOnly bool `yaml:"failedOnly" json:"-"`
+	// FailedClassesOnly has been deprecated, use FailedOnly
```

Comment:
You can elicit some help from the IDE if the term _deprecated_ is at the beginning of the sentence.

See right now:
![Screenshot 2023-04-26 at 9 53 59 AM](https://user-images.githubusercontent.com/1869292/234647758-3b870a73-584f-4f4d-bae4-19e0494fdcdf.png)

Versus:
![Screenshot 2023-04-26 at 9 59 14 AM](https://user-images.githubusercontent.com/1869292/234648882-fb7de46d-5224-49d7-8122-f8b7c25bda3d.png)

Reply:
Ah nice!